packages/simulator/streams/orders/hm.js

```diff
@@ -86,12 +86,20 @@ function read() {
       )
     ),
     mergeMap(({ key, rows }) => {
+      // TODO: Figure out a good way to distribute the orders to the distribution centers.
```

Comment:
Egentligen skulle vi mata in fleets.json här så att paketen kan fördelas enligt flottornas inbördes marknadsandelar.